.github/workflows/stack-linter.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       # Run Linter against code base #
       ################################
       - name: Lint Code Base
-        uses: docker://github/super-linter:latest
+        uses: github/super-linter:latest
```

Comment:
Why are we changing this? I thought that using the docker image was way faster. @admiralAwkbar

Reply:
I see your note at https://github.com/github/super-linter/pull/577. I understand now that these are the same thing.